features/features.go

```diff
@@ -670,4 +670,12 @@ var (
 						enableIn(configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 						enhancementPR("https://github.com/openshift/enhancements/pull/1697").
 						mustRegister()
+
+	FeatureGateNutanixMultiSubnets = newFeatureGate("NutanixMultiSubnets").
+					reportProblemsToJiraComponent("SPLAT").
+					contactPerson("yanhli").
+					productScope(ocpSpecific).
+					enhancementPR(legacyFeatureGateWithoutEnhancement).
```

Comment:
Links to enhancement PRs are required for new FeatureGates.  We need to know what we're trying to accomplish, how we will accomplish it, and how we'll know we're finished.  https://github.com/openshift/enhancements is a place to describe that.

Reply:
changed with https://github.com/openshift/enhancements/pull/1711.